hybrid-iam-ra-setup.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+exec >&2
+set -eux -o pipefail
+
+declare -r USER_DATA="/.bottlerocket/host-containers/current/user-data"
+declare -r SECRETS_DIR="/.bottlerocket/rootfs/var/lib/eks-hybrid/roles-anywhere/pki/"
+
+NODE_CERT_DATA=$(jq -r '.certificate' $USER_DATA)
+NODE_KEY_DATA=$(jq -r '.key' $USER_DATA)
```

Comment:
Instead the script should be invoked like:
```
eks-hybrid-iam-ra-setup --key "${KEY}" --certificate "${CERTIFICATE}"
```

And it should parse args at the top to get those values instead of using `jq`.

[rpm2kmodkit](https://github.com/bottlerocket-os/twoliter/blob/14a43adf49de216056d069199f17c1519f59489a/twoliter/embedded/rpm2kit#L8) has an example of a simple arg parsing loop in shell that should be sufficient here.